ert3_examples/snake_oil/run_demo

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# This script serves both as an example, and for CI purposes
+
+set -e
+
+ert3 clean --all  # Only relevant when run manually
+
+# Let ERT draw and run a stochastic ensemble:
+ert3 run evaluation
+ert3 export evaluation
+# Assert that we have FOPT in the output
+grep FOPT experiments/evaluation/data.json >/dev/null
+
+
+# DOE (design of experiment)
+design2params 0 resources/designmatrix.xlsx designsheet defaults  # [semeio]
```

Comment:
ok, so this actually handles the `xlrd` issue. Just confirming that it needs to be there :)

Reply:
Necessary for producing the json files yes :)